cpp/dolfinx/fem/Function.h

```diff
@@ -373,37 +398,53 @@ class Function
       cell_info = std::span(mesh->topology().get_cell_permutation_info());
     }
 
-    xt::xtensor<double, 2> coordinate_dofs
-        = xt::zeros<double>({num_dofs_g, gdim});
-    xt::xtensor<double, 2> xp = xt::zeros<double>({std::size_t(1), gdim});
+    namespace stdex = std::experimental;
+    using cmdspan4_t
+        = stdex::mdspan<const double, stdex::dextents<std::size_t, 4>>;
+    using mdspan2_t = stdex::mdspan<double, stdex::dextents<std::size_t, 2>>;
+    using mdspan3_t = stdex::mdspan<double, stdex::dextents<std::size_t, 3>>;
+
+    std::vector<double> coord_dofs_b(num_dofs_g * gdim);
```

Comment:
General question, what does the `*_b` stand for when we create memory allocations for the spans?

Reply:
'buffer'